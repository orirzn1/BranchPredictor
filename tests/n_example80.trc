2 7 26 0 global_history local_tables using_share_lsb
0x8dffa08 T 0xd0230
0x8dffa08 N 0xd0230
0x324935c8 N 0x77be0198
0x324935c8 T 0x77be0198
0x324935c8 N 0x77be0198
0x0940c N 0x26b0c
0x965df48 T 0x0b297c
0x8dffa08 T 0xd0230
0x965df48 T 0x0b297c
0x8dffa08 T 0xd0230
0x965df48 T 0x0b297c
0x8dffa08 N 0xd0230
0x324935c8 N 0x77be0198
0x139e0 T 0x09f4
0x965df48 T 0x0b297c
0x965df48 T 0x0b297c
0x965df48 T 0x0b297c
0x54b8 T 0xc04e658c
0x8dffa08 T 0xd0230
0x8dffa08 N 0xd0230
0x54b8 N 0xc04e658c
0x139e0 T 0x09f4
0x139e0 N 0x09f4
0x0940c N 0x26b0c
0x139e0 T 0x09f4
0x54b8 T 0xc04e658c
0x965df48 N 0x0b297c
0x54b8 N 0xc04e658c
0x0940c N 0x26b0c
0x324935c8 N 0x77be0198
0x54b8 N 0xc04e658c
0x324935c8 N 0x77be0198
0x54b8 T 0xc04e658c
0x324935c8 T 0x77be0198
0x139e0 N 0x09f4
0x8dffa08 N 0xd0230
0x8dffa08 T 0xd0230
0x139e0 N 0x09f4
0x0940c N 0x26b0c
0x54b8 T 0xc04e658c
0x54b8 T 0xc04e658c
0x965df48 T 0x0b297c
0x965df48 N 0x0b297c
0x0940c T 0x26b0c
0x8dffa08 N 0xd0230
0x965df48 T 0x0b297c
0x139e0 N 0x09f4
0x0940c N 0x26b0c
0x324935c8 T 0x77be0198
0x8dffa08 T 0xd0230
0x54b8 T 0xc04e658c
0x0940c T 0x26b0c
0x139e0 N 0x09f4
0x54b8 T 0xc04e658c
0x324935c8 T 0x77be0198
0x0940c T 0x26b0c
0x965df48 N 0x0b297c
0x965df48 N 0x0b297c
0x8dffa08 T 0xd0230
0x0940c N 0x26b0c
0x54b8 T 0xc04e658c
0x965df48 N 0x0b297c
0x8dffa08 T 0xd0230
0x139e0 T 0x09f4
0x324935c8 N 0x77be0198
0x965df48 N 0x0b297c
0x54b8 T 0xc04e658c
0x0940c N 0x26b0c
0x965df48 N 0x0b297c
0x324935c8 T 0x77be0198
0x965df48 T 0x0b297c
0x324935c8 T 0x77be0198
0x54b8 N 0xc04e658c
0x139e0 N 0x09f4
0x54b8 T 0xc04e658c
0x0940c N 0x26b0c
0x139e0 T 0x09f4
0x139e0 T 0x09f4
0x0940c N 0x26b0c
0x54b8 T 0xc04e658c
0x139e0 N 0x09f4
0x8dffa08 T 0xd0230
0x965df48 T 0x0b297c
0x324935c8 T 0x77be0198
0x8dffa08 T 0xd0230
0x139e0 N 0x09f4
0x0940c N 0x26b0c
0x139e0 T 0x09f4
0x139e0 N 0x09f4
0x324935c8 N 0x77be0198
0x965df48 T 0x0b297c
0x0940c N 0x26b0c
0x0940c N 0x26b0c
0x324935c8 N 0x77be0198
0x965df48 N 0x0b297c
0x965df48 T 0x0b297c
0x0940c N 0x26b0c
0x54b8 T 0xc04e658c
0x8dffa08 T 0xd0230
0x0940c N 0x26b0c
0x0940c T 0x26b0c
0x8dffa08 T 0xd0230
0x54b8 T 0xc04e658c
0x965df48 T 0x0b297c
0x8dffa08 N 0xd0230
0x0940c T 0x26b0c
0x54b8 N 0xc04e658c
0x54b8 T 0xc04e658c
0x0940c T 0x26b0c
0x8dffa08 T 0xd0230
0x324935c8 N 0x77be0198
0x965df48 T 0x0b297c
0x324935c8 T 0x77be0198
0x324935c8 T 0x77be0198
0x324935c8 N 0x77be0198
0x8dffa08 N 0xd0230
0x139e0 T 0x09f4
0x139e0 T 0x09f4
0x8dffa08 T 0xd0230
0x139e0 T 0x09f4
0x0940c T 0x26b0c
0x8dffa08 N 0xd0230
0x965df48 T 0x0b297c
0x139e0 N 0x09f4
0x0940c N 0x26b0c
0x324935c8 T 0x77be0198
0x965df48 T 0x0b297c
0x8dffa08 N 0xd0230
0x139e0 N 0x09f4
0x8dffa08 T 0xd0230
0x8dffa08 N 0xd0230
0x8dffa08 N 0xd0230
0x324935c8 T 0x77be0198
0x0940c T 0x26b0c
0x8dffa08 T 0xd0230
0x965df48 T 0x0b297c
0x139e0 T 0x09f4
0x54b8 T 0xc04e658c
0x8dffa08 T 0xd0230
0x54b8 N 0xc04e658c
0x8dffa08 T 0xd0230
0x8dffa08 T 0xd0230
0x54b8 T 0xc04e658c
0x54b8 T 0xc04e658c
0x324935c8 T 0x77be0198
0x8dffa08 N 0xd0230
0x54b8 T 0xc04e658c
0x8dffa08 T 0xd0230
0x324935c8 T 0x77be0198
0x139e0 T 0x09f4
0x54b8 T 0xc04e658c
0x324935c8 T 0x77be0198
0x139e0 N 0x09f4
0x324935c8 N 0x77be0198
0x0940c N 0x26b0c
0x0940c T 0x26b0c
0x54b8 T 0xc04e658c
0x324935c8 T 0x77be0198
0x139e0 T 0x09f4
0x0940c T 0x26b0c
0x54b8 N 0xc04e658c
0x139e0 T 0x09f4
0x54b8 T 0xc04e658c
0x0940c N 0x26b0c
0x965df48 N 0x0b297c
0x324935c8 T 0x77be0198
0x139e0 N 0x09f4
0x0940c T 0x26b0c
0x8dffa08 N 0xd0230
0x324935c8 T 0x77be0198
0x0940c T 0x26b0c
0x0940c T 0x26b0c
0x324935c8 T 0x77be0198
0x54b8 T 0xc04e658c
0x54b8 N 0xc04e658c
0x139e0 N 0x09f4
0x8dffa08 T 0xd0230
0x0940c T 0x26b0c
0x965df48 N 0x0b297c
0x54b8 N 0xc04e658c
0x0940c N 0x26b0c
0x0940c N 0x26b0c
0x0940c N 0x26b0c
0x54b8 N 0xc04e658c
0x54b8 N 0xc04e658c
0x139e0 N 0x09f4
0x965df48 N 0x0b297c
0x324935c8 N 0x77be0198
0x8dffa08 T 0xd0230
0x139e0 N 0x09f4
0x965df48 N 0x0b297c
0x8dffa08 T 0xd0230
0x54b8 T 0xc04e658c
0x0940c N 0x26b0c
0x324935c8 T 0x77be0198
0x8dffa08 N 0xd0230
0x324935c8 N 0x77be0198
0x0940c T 0x26b0c
0x54b8 N 0xc04e658c
0x54b8 T 0xc04e658c
0x965df48 N 0x0b297c
0x965df48 T 0x0b297c
0x139e0 T 0x09f4
0x8dffa08 N 0xd0230
0x324935c8 T 0x77be0198
0x8dffa08 N 0xd0230
0x54b8 N 0xc04e658c
0x139e0 N 0x09f4
0x324935c8 N 0x77be0198
0x0940c N 0x26b0c
0x139e0 N 0x09f4
0x0940c N 0x26b0c
0x139e0 T 0x09f4
0x54b8 T 0xc04e658c
0x8dffa08 N 0xd0230
0x54b8 T 0xc04e658c
0x0940c T 0x26b0c
0x0940c N 0x26b0c
0x8dffa08 T 0xd0230
0x965df48 N 0x0b297c
0x0940c N 0x26b0c
0x324935c8 N 0x77be0198
0x54b8 T 0xc04e658c
0x0940c N 0x26b0c
0x0940c N 0x26b0c
0x54b8 N 0xc04e658c
0x8dffa08 N 0xd0230
0x965df48 N 0x0b297c
0x54b8 T 0xc04e658c
0x324935c8 N 0x77be0198
0x0940c T 0x26b0c
0x324935c8 T 0x77be0198
0x54b8 T 0xc04e658c
0x8dffa08 N 0xd0230
0x0940c T 0x26b0c
0x965df48 N 0x0b297c
0x54b8 N 0xc04e658c
0x8dffa08 T 0xd0230
0x8dffa08 T 0xd0230
0x324935c8 N 0x77be0198
0x324935c8 T 0x77be0198
0x54b8 N 0xc04e658c
0x8dffa08 T 0xd0230
0x8dffa08 T 0xd0230
0x8dffa08 N 0xd0230
0x54b8 N 0xc04e658c
0x324935c8 N 0x77be0198
0x965df48 N 0x0b297c
0x139e0 T 0x09f4
0x54b8 N 0xc04e658c
0x8dffa08 T 0xd0230
0x965df48 T 0x0b297c
0x324935c8 N 0x77be0198
0x8dffa08 T 0xd0230
0x54b8 T 0xc04e658c
0x324935c8 T 0x77be0198
0x8dffa08 T 0xd0230
0x139e0 T 0x09f4
0x139e0 N 0x09f4
0x0940c T 0x26b0c
0x324935c8 N 0x77be0198
0x8dffa08 N 0xd0230
0x54b8 T 0xc04e658c
0x8dffa08 T 0xd0230
0x8dffa08 T 0xd0230
0x324935c8 T 0x77be0198
0x0940c N 0x26b0c
0x8dffa08 N 0xd0230
0x139e0 N 0x09f4
0x0940c T 0x26b0c
0x0940c N 0x26b0c
0x8dffa08 T 0xd0230
0x8dffa08 N 0xd0230
0x965df48 T 0x0b297c
0x8dffa08 N 0xd0230
0x54b8 T 0xc04e658c
0x965df48 N 0x0b297c
0x0940c N 0x26b0c
0x139e0 N 0x09f4
0x8dffa08 N 0xd0230
0x54b8 T 0xc04e658c
0x8dffa08 T 0xd0230
0x0940c T 0x26b0c
0x965df48 T 0x0b297c
0x8dffa08 T 0xd0230
0x8dffa08 N 0xd0230
0x0940c N 0x26b0c
0x965df48 N 0x0b297c
0x324935c8 N 0x77be0198
0x965df48 N 0x0b297c
0x54b8 N 0xc04e658c
0x965df48 N 0x0b297c
0x54b8 T 0xc04e658c
0x324935c8 T 0x77be0198
0x54b8 N 0xc04e658c
0x0940c T 0x26b0c
0x8dffa08 T 0xd0230
0x8dffa08 N 0xd0230
0x139e0 N 0x09f4
0x965df48 T 0x0b297c
0x8dffa08 N 0xd0230
0x8dffa08 T 0xd0230
0x0940c N 0x26b0c
0x139e0 T 0x09f4
0x139e0 N 0x09f4
0x0940c N 0x26b0c
0x965df48 N 0x0b297c
0x139e0 N 0x09f4
0x965df48 N 0x0b297c
0x139e0 T 0x09f4
0x54b8 N 0xc04e658c
0x965df48 N 0x0b297c
0x965df48 T 0x0b297c
0x139e0 N 0x09f4
0x324935c8 T 0x77be0198
0x8dffa08 T 0xd0230
0x0940c T 0x26b0c
0x8dffa08 N 0xd0230
0x8dffa08 T 0xd0230