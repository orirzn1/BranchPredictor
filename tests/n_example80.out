0x8dffa08 N 0x8dffa0c
0x8dffa08 N 0x8dffa0c
0x324935c8 N 0x324935cc
0x324935c8 N 0x324935cc
0x324935c8 N 0x324935cc
0x940c N 0x9410
0x965df48 N 0x965df4c
0x8dffa08 N 0x8dffa0c
0x965df48 N 0x965df4c
0x8dffa08 N 0x8dffa0c
0x965df48 N 0x965df4c
0x8dffa08 N 0x8dffa0c
0x324935c8 N 0x324935cc
0x139e0 N 0x139e4
0x965df48 N 0x965df4c
0x965df48 N 0x965df4c
0x965df48 N 0x965df4c
0x54b8 N 0x54bc
0x8dffa08 N 0x8dffa0c
0x8dffa08 N 0x8dffa0c
0x54b8 N 0x54bc
0x139e0 N 0x139e4
0x139e0 N 0x139e4
0x940c N 0x9410
0x139e0 N 0x139e4
0x54b8 N 0x54bc
0x965df48 N 0x965df4c
0x54b8 N 0x54bc
0x940c N 0x9410
0x324935c8 N 0x324935cc
0x54b8 N 0x54bc
0x324935c8 N 0x324935cc
0x54b8 N 0x54bc
0x324935c8 N 0x324935cc
0x139e0 N 0x139e4
0x8dffa08 N 0x8dffa0c
0x8dffa08 N 0x8dffa0c
0x139e0 N 0x139e4
0x940c N 0x9410
0x54b8 N 0x54bc
0x54b8 N 0x54bc
0x965df48 N 0x965df4c
0x965df48 N 0x965df4c
0x940c N 0x9410
0x8dffa08 N 0x8dffa0c
0x965df48 N 0x965df4c
0x139e0 N 0x139e4
0x940c N 0x9410
0x324935c8 N 0x324935cc
0x8dffa08 N 0x8dffa0c
0x54b8 N 0x54bc
0x940c N 0x9410
0x139e0 N 0x139e4
0x54b8 N 0x54bc
0x324935c8 N 0x324935cc
0x940c N 0x9410
0x965df48 N 0x965df4c
0x965df48 N 0x965df4c
0x8dffa08 N 0x8dffa0c
0x940c N 0x9410
0x54b8 N 0x54bc
0x965df48 N 0x965df4c
0x8dffa08 N 0x8dffa0c
0x139e0 N 0x139e4
0x324935c8 N 0x324935cc
0x965df48 N 0x965df4c
0x54b8 N 0x54bc
0x940c N 0x9410
0x965df48 N 0x965df4c
0x324935c8 N 0x324935cc
0x965df48 N 0x965df4c
0x324935c8 N 0x324935cc
0x54b8 N 0x54bc
0x139e0 N 0x139e4
0x54b8 N 0x54bc
0x940c N 0x9410
0x139e0 N 0x139e4
0x139e0 N 0x139e4
0x940c N 0x9410
0x54b8 N 0x54bc
0x139e0 N 0x139e4
0x8dffa08 N 0x8dffa0c
0x965df48 N 0x965df4c
0x324935c8 N 0x324935cc
0x8dffa08 N 0x8dffa0c
0x139e0 N 0x139e4
0x940c N 0x9410
0x139e0 N 0x139e4
0x139e0 N 0x139e4
0x324935c8 N 0x324935cc
0x965df48 N 0x965df4c
0x940c N 0x9410
0x940c N 0x9410
0x324935c8 N 0x324935cc
0x965df48 N 0x965df4c
0x965df48 N 0x965df4c
0x940c N 0x9410
0x54b8 N 0x54bc
0x8dffa08 N 0x8dffa0c
0x940c N 0x9410
0x940c N 0x9410
0x8dffa08 N 0x8dffa0c
0x54b8 N 0x54bc
0x965df48 N 0x965df4c
0x8dffa08 N 0x8dffa0c
0x940c N 0x9410
0x54b8 N 0x54bc
0x54b8 N 0x54bc
0x940c N 0x9410
0x8dffa08 N 0x8dffa0c
0x324935c8 N 0x324935cc
0x965df48 N 0x965df4c
0x324935c8 N 0x324935cc
0x324935c8 N 0x324935cc
0x324935c8 N 0x324935cc
0x8dffa08 N 0x8dffa0c
0x139e0 N 0x139e4
0x139e0 N 0x139e4
0x8dffa08 N 0x8dffa0c
0x139e0 N 0x139e4
0x940c N 0x9410
0x8dffa08 N 0x8dffa0c
0x965df48 N 0x965df4c
0x139e0 N 0x139e4
0x940c N 0x9410
0x324935c8 N 0x324935cc
0x965df48 N 0x965df4c
0x8dffa08 N 0x8dffa0c
0x139e0 N 0x139e4
0x8dffa08 N 0x8dffa0c
0x8dffa08 N 0x8dffa0c
0x8dffa08 N 0x8dffa0c
0x324935c8 N 0x324935cc
0x940c N 0x9410
0x8dffa08 N 0x8dffa0c
0x965df48 N 0x965df4c
0x139e0 N 0x139e4
0x54b8 N 0x54bc
0x8dffa08 N 0x8dffa0c
0x54b8 N 0x54bc
0x8dffa08 N 0x8dffa0c
0x8dffa08 N 0x8dffa0c
0x54b8 N 0x54bc
0x54b8 N 0x54bc
0x324935c8 N 0x324935cc
0x8dffa08 N 0x8dffa0c
0x54b8 N 0x54bc
0x8dffa08 N 0x8dffa0c
0x324935c8 N 0x324935cc
0x139e0 N 0x139e4
0x54b8 N 0x54bc
0x324935c8 N 0x324935cc
0x139e0 N 0x139e4
0x324935c8 N 0x324935cc
0x940c N 0x9410
0x940c N 0x9410
0x54b8 N 0x54bc
0x324935c8 N 0x324935cc
0x139e0 N 0x139e4
0x940c N 0x9410
0x54b8 N 0x54bc
0x139e0 N 0x139e4
0x54b8 N 0x54bc
0x940c N 0x9410
0x965df48 N 0x965df4c
0x324935c8 N 0x324935cc
0x139e0 N 0x139e4
0x940c N 0x9410
0x8dffa08 N 0x8dffa0c
0x324935c8 N 0x324935cc
0x940c N 0x9410
0x940c N 0x9410
0x324935c8 N 0x324935cc
0x54b8 N 0x54bc
0x54b8 N 0x54bc
0x139e0 N 0x139e4
0x8dffa08 N 0x8dffa0c
0x940c N 0x9410
0x965df48 N 0x965df4c
0x54b8 N 0x54bc
0x940c N 0x9410
0x940c N 0x9410
0x940c N 0x9410
0x54b8 N 0x54bc
0x54b8 N 0x54bc
0x139e0 N 0x139e4
0x965df48 N 0x965df4c
0x324935c8 N 0x324935cc
0x8dffa08 N 0x8dffa0c
0x139e0 N 0x139e4
0x965df48 N 0x965df4c
0x8dffa08 N 0x8dffa0c
0x54b8 N 0x54bc
0x940c N 0x9410
0x324935c8 N 0x324935cc
0x8dffa08 N 0x8dffa0c
0x324935c8 N 0x324935cc
0x940c N 0x9410
0x54b8 N 0x54bc
0x54b8 N 0x54bc
0x965df48 N 0x965df4c
0x965df48 N 0x965df4c
0x139e0 N 0x139e4
0x8dffa08 N 0x8dffa0c
0x324935c8 N 0x324935cc
0x8dffa08 N 0x8dffa0c
0x54b8 N 0x54bc
0x139e0 N 0x139e4
0x324935c8 N 0x324935cc
0x940c N 0x9410
0x139e0 N 0x139e4
0x940c N 0x9410
0x139e0 N 0x139e4
0x54b8 N 0x54bc
0x8dffa08 N 0x8dffa0c
0x54b8 N 0x54bc
0x940c N 0x9410
0x940c N 0x9410
0x8dffa08 N 0x8dffa0c
0x965df48 N 0x965df4c
0x940c N 0x9410
0x324935c8 N 0x324935cc
0x54b8 N 0x54bc
0x940c N 0x9410
0x940c N 0x9410
0x54b8 N 0x54bc
0x8dffa08 N 0x8dffa0c
0x965df48 N 0x965df4c
0x54b8 N 0x54bc
0x324935c8 N 0x324935cc
0x940c N 0x9410
0x324935c8 N 0x324935cc
0x54b8 N 0x54bc
0x8dffa08 N 0x8dffa0c
0x940c N 0x9410
0x965df48 N 0x965df4c
0x54b8 N 0x54bc
0x8dffa08 N 0x8dffa0c
0x8dffa08 N 0x8dffa0c
0x324935c8 N 0x324935cc
0x324935c8 N 0x324935cc
0x54b8 N 0x54bc
0x8dffa08 N 0x8dffa0c
0x8dffa08 N 0x8dffa0c
0x8dffa08 N 0x8dffa0c
0x54b8 N 0x54bc
0x324935c8 N 0x324935cc
0x965df48 N 0x965df4c
0x139e0 N 0x139e4
0x54b8 N 0x54bc
0x8dffa08 N 0x8dffa0c
0x965df48 N 0x965df4c
0x324935c8 N 0x324935cc
0x8dffa08 N 0x8dffa0c
0x54b8 N 0x54bc
0x324935c8 N 0x324935cc
0x8dffa08 N 0x8dffa0c
0x139e0 N 0x139e4
0x139e0 N 0x139e4
0x940c N 0x9410
0x324935c8 N 0x324935cc
0x8dffa08 N 0x8dffa0c
0x54b8 N 0x54bc
0x8dffa08 N 0x8dffa0c
0x8dffa08 N 0x8dffa0c
0x324935c8 N 0x324935cc
0x940c N 0x9410
0x8dffa08 N 0x8dffa0c
0x139e0 N 0x139e4
0x940c N 0x9410
0x940c N 0x9410
0x8dffa08 N 0x8dffa0c
0x8dffa08 N 0x8dffa0c
0x965df48 N 0x965df4c
0x8dffa08 N 0x8dffa0c
0x54b8 N 0x54bc
0x965df48 N 0x965df4c
0x940c N 0x9410
0x139e0 N 0x139e4
0x8dffa08 N 0x8dffa0c
0x54b8 N 0x54bc
0x8dffa08 N 0x8dffa0c
0x940c N 0x9410
0x965df48 N 0x965df4c
0x8dffa08 N 0x8dffa0c
0x8dffa08 N 0x8dffa0c
0x940c N 0x9410
0x965df48 N 0x965df4c
0x324935c8 N 0x324935cc
0x965df48 N 0x965df4c
0x54b8 N 0x54bc
0x965df48 N 0x965df4c
0x54b8 N 0x54bc
0x324935c8 N 0x324935cc
0x54b8 N 0x54bc
0x940c N 0x9410
0x8dffa08 N 0x8dffa0c
0x8dffa08 N 0x8dffa0c
0x139e0 N 0x139e4
0x965df48 N 0x965df4c
0x8dffa08 N 0x8dffa0c
0x8dffa08 N 0x8dffa0c
0x940c N 0x9410
0x139e0 N 0x139e4
0x139e0 N 0x139e4
0x940c N 0x9410
0x965df48 N 0x965df4c
0x139e0 N 0x139e4
0x965df48 N 0x965df4c
0x139e0 N 0x139e4
0x54b8 N 0x54bc
0x965df48 N 0x965df4c
0x965df48 N 0x965df4c
0x139e0 N 0x139e4
0x324935c8 N 0x324935cc
0x8dffa08 N 0x8dffa0c
0x940c N 0x9410
0x8dffa08 N 0x8dffa0c
0x8dffa08 N 0x8dffa0c
flush_num: 166, br_num: 319, size: 633b
